regen/feature.pl

```diff
@@ -77,7 +81,7 @@ BEGIN
 my %feature_bundle = (
     all     => [ sort keys %feature ],
     default => [ qw{indirect multidimensional bareword_filehandles
-                    apostrophe_as_package_separator} ],
+                    apostrophe_as_package_separator smartmatch} ],
```

Comment:
Actually, since smartmatch was added in v5.10 (or v5.9.5), it shouldn't be in the default bundle.

Reply:
Existing code could be using smartmatch without a use vNN.

Reply:
Oh right. I was confused by the `switch` feature, sorry.